client/src/i18n/en/shipment.json

```diff
@@ -0,0 +1,44 @@
+{
+  "SHIPMENT" : {
+    "TITLE" : "Shipment",
+    "SHIPMENTS": "Shipments",
+    "SHIPPER": "Shipper",
+    "REQUISITION": "Requisition",
+    "ORIGIN_DEPOT": "Origin depot",
+    "CURRENT_DEPOT": "Current depot",
+    "CURRENT_LOCATION": "Current location",
+    "DESTINATION_DEPOT": "Destination depot",
+    "DATE_PACKED": "Date packed",
+    "DATE_SENT": "Date sent",
+    "DATE_DELIVERED": "Date delivered",
+    "ANTICIPATED_DELIVERY_DATE": "Anticipated delivery date",
+    "RECEIVER": "Receiver",
+    "CREATED_BY": "Created by",
+    "CREATE": "Create",
+    "EDIT": "Edit",
+    "DELETE": "Delete",
+    "REFERENCE": "Reference",
+    "STOCK_REFERENCE": "Stock Reference",
+    "PACKING": "Packing",
+    "PACKING_LIST": "Packing List",
+    "NEW_SHIPMENT": "New Shipment",
+    "SHIPMENT_REGISTRY": "Shipment Registry",
+    "UPDATE_LOCATION": "Update Location",
+    "UPDATE_LOCATION_MESSAGE": "You are about to change the location of this shipment, this information will be saved in the history of this shipment",
+    "UPDATE_LOCATION_WARNING": "You cannot change the location because only shipments with an IN-TRANSIT status can benefit from the change of location",
```

Comment:
Suggest rewording.  In "only shipments with an IN-TRANSIT status can benefit from a change in location" change "benefit" to "allow".